src/main/bluesky/paginator.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+
+df = pd.read_csv('src/main/bluesky/Data/complete.csv')
+df.fillna(value=pd.NA, inplace=True)
+
+
+def paginate_dataframe(page_size, page_number, sortby):
+    # Check if the specified column exists in the DataFrame
+    var = "CALLSIGN"
+    if sortby in df.columns:
+        var = sortby
+    full = df.sort_values(by=var)
+    start_index = (page_number - 1) * page_size
+    end_index = start_index + page_size
+    page_data = full.iloc[start_index:end_index]
+
+    # Convert the page_data to CSV (assuming you want CSV format)
+    page_data_csv = page_data.to_csv(index=False)
+
```

Comment:
This depends on the implementation again, Right now you return a CSV format, Should you maybe not specify what type of pagination has happened on this file so a FE user knows what paginated request gets returned

```
{
pageSize,
TotalPages,
SortedBy,
pageNumber,
Page_data_csv
}
```

Something like above?